core/src/components/toast/toast.tsx

```diff
@@ -281,6 +287,10 @@ export class Toast implements ComponentInterface, OverlayInterface {
           <div class="toast-container" part="container">
             {this.renderButtons(startButtons, 'start')}
 
+            {this.icon !== undefined &&
+              <ion-icon class="toast-icon" part="icon" icon={this.icon}></ion-icon>
```

Comment:
We need to document this shadow part up top. See above the `@Component` decorator. We need to add something like this:

```
@part icon - The icon that appears next to the toast content.
```